src/Raven.Server/Documents/ETL/Providers/AI/Embeddings/Test/TestEmbeddingsGenerationScript.cs

```diff
@@ -0,0 +1,9 @@
+﻿using Raven.Client.Documents.Operations.AI;
+using Raven.Server.Documents.ETL.Test;
+
+namespace Raven.Server.Documents.ETL.Providers.AI.Embeddings.Test;
+
+public sealed class TestEmbeddingsGenerationScript : TestEtlScript<EmbeddingsGenerationConfiguration, AiConnectionString>
```

Comment:
What value does this give us?

Reply:
by design, this is fine

Reply:
Note that testing functionality isn't actually implemented so it might be changed - https://issues.hibernatingrhinos.com/issue/RavenDB-23834/AI-Integration-Testing-functionality-of-embedding-paths-script